pygn_handler.py

```diff
@@ -74,20 +121,15 @@ def generate_argparser():
 ### main
 ###
 
-# todo: breaks pixel configuration
-callbacks = {
-    "1": lambda board: board.fen(),
-    "2": lambda board: chess.svg.board(board=board),
-}
 
 if __name__ == '__main__':
     signal.signal(signal.SIGPIPE, signal.SIG_DFL)
 
-    argparser = generate_argparser()
```

Comment:
I was getting complaints about `argparser` conflicting with declaration in `generate_argparser()`. Capitalization of global namespace variables is a PEP8 thing, anyway